### DISCUSSION TITLE
worrying unknown portainer containers that seem attached to frigate

### DISCUSSION THREAD

**Discussion by stain3565** [blakeblackshear/frigate#4466] [General]
I have noticed (and removed) some containers within portainer on a raspberry pi4, that exited when I stopped my frigate container. One was called pensive_murdoch and one goofy_mcnulty.

I don't know where these came from but I did have an issue some time ago where portainer completely lost/forgot the associated web ui password and I did run a helper that seems to be suggested in many places on the web (helper-reset-password). Maybe that has installed some malware. I realise this could be a portainer issue but the link to frigate stopping did suggest possible connections. Anyone had similar issues? Do I need to reset my rpi4?

**Comment by NickM-27** [comment]
I wouldn't worry about it being malicious, those are default container names that docker creates. How did you install frigate, via docker compose in portainer?

**Reply by stain3565** [reply]
yep

**Reply by NickM-27** [reply]
Here's any example

![Screen Shot 2022-11-21 at 14 46 38 PM](https://user-images.githubusercontent.com/14866235/203164524-97f1d4d4-f24f-4e5a-b4fc-e82fc1f50d10.png)

if you see them again it will be helpful if you don't remove them so you can figure out what they were installed with

**Reply by stain3565** [reply]
ok thanks. Sounds like they may not be an issue but I will look out for them/
